### ISSUE TITLE
Synopsis uses Test::Alien rather than Test::Alien::CPP

### ISSUE THREAD

**Issue by djerius** [plicease/Test-Alien-CPP#10]
Howdy!

A minor typo in the Synopsis section in the docs.

It should say `use Test::Alien::CPP;`

**Comment by plicease** [comment]
Agreed!  #11 and 1.03 (on its way to CPAN) should address this.  Thank you for reporting this!
